pbsmetrics/prometheus/prometheus.go

```diff
@@ -187,6 +188,12 @@ func NewMetrics(cfg config.PrometheusMetrics) *Metrics {
 		"Count of total requests to Prebid Server labeled by account.",
 		[]string{accountLabel})
 
+	metrics.requestsQueueTimer = newHistogram(cfg, metrics.Registry,
```

Comment:
Please also add to preload.go to reduce runtime overhead and to allow the gatekeeping tests to function properly.

Reply:
Good catch, added